sample-apps/communication/main.js

```diff
@@ -84,6 +87,10 @@ function onNavigateUsingMessageClick ({ skipPreview = false } = {}) {
     packageName: communicationSamplePackageName,
     actions: { navigationSuccess: true, navigationCancelled: true }
   }
+  if (noaccessibility) {
+    // To not show the accessibility options menu, send a preset option (e.g. 'all' for 'Fastest')
+    data.presetCapability = 'all'
```

Comment:
Why is `all` related to `fastest`?

Reply:
if we allow all possibilities - we get the fastest way

Reply:
When all options are available to you (elevator, stairs, etc.) - the system is free to choose the fastest route